e2e/AndroidUtils.js

```diff
@@ -21,5 +24,12 @@ module.exports = {
   },
   executeShellCommand: (command) => {
     exec.execSync(`adb shell ${command}`);
+  },
+  unlockPhoneByPin: (pin) => {
```

Comment:
The name is misleading - the method enters text to the currently focused InputText and presses enter - for simplicity it should enter text and that's it.

1. Please rename it it `enterText(text)`
2. It should do one thing, enter text. In your test, call `enterText` following by `pressEnter`